src/index.js

```diff
@@ -41,6 +43,8 @@ export {
   DefaultGraph,
   Quad,
   Triple,
+  escape,
+  unescape,
```

Comment:
I wouldn't expose them; they're highly internal functions for ID conversion.

Reply:
Agreed. I thought this would be useful for end-users but actually there are no realistic use cases I can think of, will solve.